jib-gradle-plugin/src/test/java/com/google/cloud/tools/jib/gradle/JibPluginTest.java

```diff
@@ -103,6 +110,22 @@ public void testCheckGradleVersion_fail() throws IOException {
     }
   }
 
+  @Test
+  public void testCheckJibVersionInvoked() {
+    Project rootProject =
+        ProjectBuilder.builder().withProjectDir(testProjectRoot.getRoot()).withName("root").build();
+    System.setProperty("jib.requiredVersion", "10000.0"); // not here yet
```

Comment:
`10000.0` is actually an invalid format, isn't it? (Doesn't start/end with brackets/parentheses.)